### DISCUSSION TITLE
Check if a single field isValid

### DISCUSSION THREAD

**Discussion by piotrkulpinski** [airjp73/rvf#228] [Q&A] [answered]
Hi, 

Is there a way to check if a given field is valid without first validating the whole form?
I know I can check `fieldErrors` but that'll be empty unless I try to submit the form first. 

I don't want to display any errors, just conditionally check validity of single field.

**Comment by airjp73** [comment, marked as answer]
I think the only way to do this without actually validating the field is to go around the library a bit rather than going through it.

```ts
const context = useFormContext();

async function checkFieldValidity() {
  // This check might not be 100% correct but should be close
  const result = await validator.validateField(context.getValues(), 'myField');
  const isValid = !!result.error;
  return isValid;
}
```

**Reply by Londeren** [reply]
From the source code:

```
/**
* @deprecated Will be removed in a future version of remix-validated-form
*/
validateField?: (unvalidatedData: GenericObject, field: string) => Promise<ValidateFieldResult>;
```

@airjp73 how can this check be made after the removing of this function?
